### ISSUE TITLE
Add user data loss description, resulting from the firmware update

### ISSUE THREAD

**Issue by szszszsz** [Nitrokey/nitrokey-documentation#49]
Add user data loss description, resulting from the firmware update

File: [storage/windows/factory-reset.rst] https://docs.nitrokey.com/storage/windows/factory-reset

**Comment by daringer** [comment]
> Add user data loss description

"Destroys Encrypted Volume" should be extended? Should it make clear that "destroy" means "data loss" ?

**Comment by szszszsz** [comment]
In this case I thought only about influence of the update process on the state of the user data (removes OTP, PWS, HV but leaves EV and OpenPGP intact).
Perhaps extending/clarifying description for "destroying data" would be a good idea.

**Comment by daringer** [comment]
oh wait, now I read again and are even more confused ...
the link above is referring to factory-reset ... but you mean update?

**Comment by szszszsz** [comment]
Yes. The factory-reset page is the only place with listed options how the user data can be removed via various operations. And what I wanted is to add to that table information, that firmware update removes the mentioned data as well in the current implementation. Do you think this is not the right place? Alternatively this can be mentioned on the update page (as firmware update is not indeed the usual part of the "working" mode).

**Comment by szszszsz** [comment]
Quick update: while testing one of the last releases it seems that after the update only the Unencrypted and Encrypted Volumes data will be retained. Password Safe, OTPs and Hidden Volumes are lost.

Details:
- https://github.com/Nitrokey/nitrokey-storage-firmware/releases/tag/V0.56
